tcms/testplans/static/testplans/js/get.js

```diff
@@ -34,8 +36,91 @@ $(document).ready(function() {
     });
 
     collapseDocumentText();
+    initTestCaseSearchAndAdd(testPlanId);
 });
 
+
+function addTestCaseToPlan(planId) {
+    const case_name = $('#search-testcase')[0].value;
+    const test_case = autocomplete_cache[case_name];
+
+    // test case is already present so don't add it
+    if ( allTestCases[test_case.id] !== undefined ) {
```

Comment:
much more elegant and JS like
```suggestion
    if (allTestCases[test_case.id]) {
```